src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -81,11 +81,11 @@
     static {
         DESC_LECTURE = new EditMeetingDescriptorBuilder().withName(VALID_NAME_LECTURE)
                 .withUrl(VALID_URL_LECTURE).withDateTime(VALID_DATETIME_LECTURE).withDuration(VALID_DURATION_LECTURE)
-                .withModule(VALID_MODULE_LECTURE).withIsRecurring(VALID_RECURRING_LECTURE).withTags(VALID_TAG_LECTURE)
+                .withModule(Index.fromZeroBased(3)).withIsRecurring(VALID_RECURRING_LECTURE).withTags(VALID_TAG_LECTURE)
```

Comment:
```suggestion
                .withModule(VALID_MODULE_INDEX_LECTURE).withIsRecurring(VALID_RECURRING_LECTURE).withTags(VALID_TAG_LECTURE)
```

For the sake of reducing potential bugs/repetitive code, perhaps we could extract this out into its own static variable?